lib/rubocop/ast/node/regexp_node.rb

```diff
@@ -21,14 +21,12 @@ def to_regexp
 
       # @return [RuboCop::AST::Node] a regopt node
       def regopt
-        first, second = *self
-        first.regopt_type? ? first : second
+        children.find(&:regopt_type?)
```

Comment:
I wonder if that is necessary. The `regopt` node is always the last child of a `regexp` node if I am not mistaken.

Reply:
In `Parser::Builders::Default`: (from parser gem)
https://github.com/whitequark/parser/blob/b73fbc0368d7170f79685907909f0ac24f79816a/lib/parser/builders/default.rb#L318
Always append `options` as `regexp`'s last child.

https://github.com/whitequark/parser/blob/b73fbc0368d7170f79685907909f0ac24f79816a/lib/parser/builders/default.rb#L1935
It also assumes `regexp`'s last child is `regopt`.

This line can be `children.last`. Acceptable?

Reply:
:+1: